### DISCUSSION TITLE
docs/ninjutsu/lost-ninjutsu/7af31f1c-eef2-42f9-b5bb-ef389d47ac94

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#3] [General]
# docs/ninjutsu/lost-ninjutsu/look-beauty-girl/

别看！！！ 会社死！！

https://996.ninja/docs/ninjutsu/lost-ninjutsu/look-beauty-girl/

**Comment by LHK5921** [comment]
你在欺骗我QAQ

**Reply by LHK5921** [reply]
鼠标长按拖拽选择“小心不要外放”下面的空白区域，好看爱看

**Reply by MingGH** [reply]
现在是2张图了
![](https://img.996.ninja/ninjutsu/286bdc866db53d87bd699fc539728200.png)

**Comment by github-guhong** [comment]
ctrl + a 查看全部

**Reply by MingGH** [reply]
上难度！图片+1
![](https://img.996.ninja/ninjutsu/93d146750c1d550a396889f82c116b42.png)

**Comment by Liumingzhao513** [comment]
找到图和视频了，但是不攒劲

**Reply by MingGH** [reply]
又更了一张，绝对攒劲，但是更难找了

**Comment by BigGrassDog** [comment]
就没有我找不到的瑟瑟！

https://img.996.ninja/ninjutsu/17136e85f087a82dfc667b4d41a125ef.jpg
https://img.996.ninja/ninjutsu/13d387578237e7db68636b54a333fb41.png
https://img.996.ninja/video%2F%E4%BA%BA%E9%97%B4%E6%9E%81%E5%93%81%E5%B0%A4%E7%89%A9-%E8%BF%99%E7%A7%8D%E8%BA%AB%E6%9D%90%E6%AF%94%E4%BE%8B%E5%A0%AA%E7%A7%B0%E5%AE%8C%E7%BE%8E.mp4
https://img.996.ninja/ninjutsu/8670007398507948658051921.jpeg

**Reply by MingGH** [reply]
虽然，你找到的速度很快，但是最后一张图找错咯。藏的图绝对是可以打开滴

**Reply by BigGrassDog** [reply]
居然用了随机数，一直刷

**Reply by MingGH** [reply]
不是随机数，当“鸡啄完了米，火烧断了锁，狗舔完面”图片就会出来的

**Reply by BigGrassDog** [reply]
不把瑟瑟交出来就跟我到下面评理去！

**Reply by MingGH** [reply]
这还只是精英怪难度，我已经想到了大罗金仙难度

**Reply by sky157865019** [reply]
斐波那契数列？

**Comment by sky157865019** [comment]
https://img.996.ninja/ninjutsu/12071947451591410028264041744568.jpeg
凑合

**Reply by MingGH** [reply]
**wooooooooow!🥳🥳🥳 奖励你这张卡片，你是第一个找到并评论的！**

<img src="https://img.996.ninja/ninjutsu/sp-successione-di-fibonacci.jpg" width="400px" />

如果你能留下邮箱（最好不常用的），我会告诉你另一个隐藏在页面上的奖励。

（2024-05-24 19:49:44已经给摸鱼党兑换）
<img src="https://img.996.ninja/ninjutsu/475a449b2137099c1e46610b026487c6.png" width="200px" />

**Reply by sky157865019** [reply]
157865019@qq.com 好人一生平安！

**Comment by q506557574** [comment]


**Reply by MingGH** [reply]
太暴力的会被ban掉

**Comment by sky157865019** [comment]
8千万认真的吗

**Reply by MingGH** [reply]
要对自己的电脑有信心

**Comment by Shawyuan2022** [comment]
不！！！！T A T 居然突然被ban了！！

**Reply by MingGH** [reply]
还有个按钮可以供大家玩耍哈哈

**Comment by TheSaltwaterRoom** [comment]
随机数的图片怎么打开啊
